.github/workflows/test.yml

```diff
@@ -0,0 +1,47 @@
+name: Test
+
+on:
+  push:
+    branches: [ master ]
```

Comment:
Why limit this to `master`?

I think it's quite useful if I can run CI on my own branches before making a PR. This way, I can already do a few iterations with CI without annoying other collaborators and create the PR only when I already know that CI is passing.

I think if you limit the branch here, this is not possible, or is it?

Reply:
I removed the restriction.